src/main/kotlin/no/nav/faktureringskomponenten/controller/mapper/ModelDtoMapperExtensions.kt

```diff
@@ -87,3 +88,8 @@ private val FakturaLinje.tilResponseDto: FakturaLinjeResponseDto
         enhetsprisPerManed = this.enhetsprisPerManed
     )
 
+val FakturamottakerRequestDto.tilFakturamottakerDto: FakturamottakerDto
+    get() = FakturamottakerDto(
+        fakturaserieReferanse = ULID.randomULID(),
```

Comment:
Tenger vi en ny ref for en oppdatering?

Reply:
Nei, er nok bare jeg som har kopiert litt raskt her.